notes/src/components/Note.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import ListItem from "@mui/material/ListItem"
+import ListItemText from "@mui/material/ListItemText"
+import Typography from '@mui/material/Typography';
+import PropTypes from "prop-types"
+import styles from "../pages/styled"
+
+function Note (props) {

```

Comment:
Arrow function should be here!
Destructure props and remove `return` statement here:
```suggestion
export const Note = ({ note }) => (
<ListItem ..>

```